Fix timestamp conversion error (#13437)
demisto/content: Packs/ApiModules/ReleaseNotes/2_2_1.md

#### Scripts
##### CSVFeedApiModule

> - Fixed an issue where the **fetch-indicators** fail when trying to convert timestamp.

Reviewer: ```suggestion
- Fixed an issue where fetching indicators failed when trying to convert a timestamp to ISO format.
```

> - Fixed an issue where fetching indicators failed when trying to convert a timestamp to ISO format.

Reviewer: ```suggestion
- Improved implementation of the timestamp conversion to ISO format.
```
